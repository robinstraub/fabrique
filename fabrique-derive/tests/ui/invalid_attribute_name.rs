use fabrique_derive::Factory;

#[derive(Factory)]
struct Anvil {
    #[fabrique(unknown_attribute = true)]
    weight: u32,
}

fn main() {}