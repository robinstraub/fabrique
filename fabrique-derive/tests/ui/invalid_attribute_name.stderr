error: Unknown field: `unknown_attribute`. Available values: `primary_key`, `referenced_key`, `relation`
 --> tests/ui/invalid_attribute_name.rs:4:1
  |
4 | struct Anvil {
  | ^^^^^^
